### PR TITLE
Fix/fengsha moist

### PR DESCRIPTION
**Pull request by bbakernoaa**
Found an issue with the gravimetric soil moisture units used within the FECAN soil moisture correction.  This fixes that as well as adds a few knobs to be able to tune the soil moisture and drylimit within the dust scheme.

### CODE REVIEW

**Comment by bbakernoaa** [comment]
Anything I need to do for this?

**Comment by mathomp4** [comment]
@bbakernoaa A couple things:

1. Can you add an entry to the CHANGELOG.md file in the root directory explaining your changes.
2. We require a label (either 0-diff or non-zero-diff) to let the @GEOS-ESM/aerosol-team team know if this would affect results. Now, technically I suppose it is non-zero-diff if you run Fengsha but then we (GMAO) don't run that I think, so...I guess you can label it zero-diff? (Since, in a way, it is to us.)

**Comment by bbakernoaa** [comment]
@mathomp4 I'll try to add something to the changelog and the label this morning.  Thanks

File: CHANGELOG.md
Comment (mathomp4) [inline CHANGELOG.md:8]:
```suggestion
## [Unreleased]
```

Comment (mathomp4) [reply CHANGELOG.md:8]: @bbakernoaa Let's keep it as unreleased. We don't know when the PR will go in and what the release number will be or the like.

**Comment by mathomp4** [comment]
> @mathomp4 I added a changelog but if I'm suppose to add a label here in the PR I don't have permission to do so.

Ah. Interesting. I didn't know you needed write access to make a label! Well, I've done it for you. I'll keep it as zero-diff since we don't run Fengsha.

But, I'll be careful and do a test to make sure!

**Comment by mathomp4** [comment]
Yep. This seems to be zero-diff at least how I know how to run GOCART (which I guess isn't Fengsha!)

**Comment by bbakernoaa** [comment]
Good this should only effect FENGSHA and not the Ginoux or K14 schemes

**Comment by tclune** [comment]
@amdasilva  @vbuchard Our NOAA colleagues would like to know when they can expect this to be merged.

**Comment by mathomp4** [comment]
@bbakernoaa We are confused by:
```
- Add `soil_moisture_factor` to the DU2G_GridComp.rc and DU2G_GridCompMod.F90 files for FENGSHA
- Add `soil_drylimit_factor` to the DU2G_GridComp.rc and DU2G_GridCompMod.F90 files for FENGSHA
```

First, do you mean `DU2G_instance_DU.rc`? Second, on your branch, I see no updates to the FENGSHA bits at the bottom:

https://github.com/bbakernoaa/GOCART/blob/fix/fengsha_moist/ESMF/GOCART2G_GridComp/DU2G_GridComp/DU2G_instance_DU.rc

there is only:
```
# FENGSHA settings
alpha: 0.3
gamma: 1.3
vertical_to_horizontal_flux_ratio_limit: 2.e-04
```

**Comment by bbakernoaa** [comment]
@mathomp4 Yes sorry.  It will need to be in the `DU2G_instance_DU.rc` file.  I used the same names as the k14 so I didn't want to duplicate it in the `rc` file. They do the same things as in the k14 scheme. 

Could you please advise on how we should do that?

**Comment by mathomp4** [comment]
> @mathomp4 Yes sorry. It will need to be in the `DU2G_instance_DU.rc` file. I used the same names as the k14 so I didn't want to duplicate it in the `rc` file. They do the same things as in the k14 scheme.
> 
> Could you please advise on how we should do that?

Ohhh. Okay. I see. I didn't notice they were above and not commented out! Well, I suppose this works then.

**Comment by vbuchard** [comment]
> @mathomp4 I just added the soil_drylimit_factor to the `.rc` file as this one wasn't duplicated

Can you please update the changelog accordingly (with the right filename) before I can approve/merge the PR. With all these updates, it looks ok to me. Thanks @mathomp4 for doing the testing.

**Comment by bbakernoaa** [comment]
@vbuchard I modified it.  Please let me know if that works.

**Review by vbuchard — APPROVED** [review]
It looks good to me. Thanks

**Pull request merged**
